site/content/community/02-roadmap.md

```diff
@@ -13,9 +13,9 @@ One or more release candidate(s) (RC) builds will be made available prior to eac
 
 ## Project Roadmap Themes
 
-### Cluster quickstart with GitOps
+### Cluster Quick Start with Gitops
```

Comment:
If we're going full lowercase then shouldn't this be the following?
```suggestion
### Cluster Quick Start with gitops
```

Reply:
hahaha I did change this one too, then cpu1 mentioned that it looks a bit weird, like all of them were capitalized except that one. I will change it again.

Reply:
Ah, yes, just saw Chetan's suggestion 😆